src/docfx/lib/markdown/LinkExtension.cs

```diff
@@ -36,6 +39,13 @@ internal static class LinkExtension
                     {
                         inline.Tag = ResolveLinks(inline.Tag, inline);
                     }
+                    else if (node is TripleColonBlock tripleColonBlock && tripleColonBlock.Extension is ImageExtension)
+                    {
+                        var imageSrc = tripleColonBlock.GetAttributes().Properties.Single(kv => kv.Key == "src").Value;
```

Comment:
nit: this algorithm is a bit in-efficient, the underlying storage of `GetAttributes().Properties` is a mutable list, using a `for` loop is more efficient and readable here.